formats/ome-tiff/file-structure.txt

```diff
@@ -0,0 +1,84 @@
+OME-TIFF file structure
+=======================
+
+The following section discusses various considerations when designing
+OME-TIFF filesets distributed over multiple files.
+
+Single versus multiple files
+----------------------------
+
+Splitting a fileset across multiple files can have advantages in terms of
+acquisition but also processing purposes. The OME-TIFF file format can support
+any image organization. However, using one TIFF file per timepoint per channel
+with the focal planes for that timepoint and channel stored sequentially
+within the TIFF makes for very easy :ref:`TiffData` elements.
```

Comment:
Pushed. As I don't have data to back up any other claim, I think easy creation is the safe call here.